tests/test_configuration.py

```diff
@@ -125,8 +125,11 @@ def config(test_config_file_path):
     environ["PREFECT__ENV_VARS__FLOAT"] = "7.5"
     environ["PREFECT__ENV_VARS__NEGATIVE_FLOAT"] = "-7.5"
     environ["PREFECT__ENV_VARS__ESCAPED_CHARACTERS"] = r"line 1\nline 2\rand 3\tand 4"
-    yield configuration.load_config_file(
-        path=test_config_file_path, env_var_prefix="PREFECT", env=environ
+
+    monkeypatch.setattr("os.environ", environ)
```

Comment:
It might be better practice here to use `monkeypatch.setenv("PREFECT__ENV_VAR", "42")` for each env var above.

Reply:
TIL